### PR TITLE
copr: fix bz#1576736

### PR DESCRIPTION
**Pull request by clime**
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1576736.

Details about name mangling: https://docs.python.org/3/reference/expressions.html?highlight=name%20mangling#atom-identifiers

### CODE REVIEW

**Comment by xsuchy** [comment]
:+1:

File: plugins/copr.py [hunk 1]
```diff
@@ -36,13 +36,13 @@
 # If that fails, attempt to import the deprecated implementation
 # from the platform module.
 try:
-    from distro import linux_distribution as __linux_distro
+    from distro import linux_distribution as linux_distro
```
Comment (m-blaha) [inline plugins/copr.py]: How about removing this `as linux_distro` import and rename the function on the line 45 to `def linux_distribution()`?

Comment (j-mracek) [reply plugins/copr.py]: @clime @xsuchy I am neutral here. What do you think?

Comment (clime) [reply plugins/copr.py]: I made the changes as requested.

**Comment by m-blaha** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**